CourseStream/CourseStream/Program.cs

```diff
@@ -48,4 +55,13 @@
 app.MapGet("/courseStreams/{id}", EndpointHandlers.GetCourseStreamById);
 app.MapPost("/courseStreams/add", EndpointHandlers.AddCourse);
 app.MapPost("courseStreams/enroll", EndpointHandlers.EnrollStudent);
+
+// Apply EF Core Migrations
+
+using (var scope = app.Services.CreateScope())
+{
+	var dbContext = scope.ServiceProvider.GetRequiredService<CourseStreamDbContext>();
+	dbContext.Database.Migrate();
+}
```

Comment:
Better to move to a separated class/method to keep entrypoint tiny

Reply:
Okay, I'll think about it, for now let's leave this working solution